appcast.xml

```diff
@@ -3,12 +3,12 @@
     <channel>
         <title>Internxt Drive</title>
         <item>
-            <title>2.1.0</title>
-            <pubDate>Mon, 27 Nov 2023 11:16:59 +0100</pubDate>
-            <sparkle:version>37</sparkle:version>
-            <sparkle:shortVersionString>2.1.0</sparkle:shortVersionString>
+            <title>2.1.1</title>
```

Comment:
Don't commit this file, or it will push a new remote update, just sync this file with the `main` branch one please